src/main.rs

```diff
@@ -64,6 +68,7 @@ fn main() -> ! {
 
     // Set up the LED (C13).
     let mut led = gpioc.pc13.into_push_pull_output();
+    led.set_high().unwrap();
```

Comment:
do you want to create a simple `Led` wrapper? `set_high()` is to turn off, I suppose, but that's confusing at glance

Reply:
I'll do this in a followup PR.